projects/plugins/jetpack/_inc/client/state/recommendations/reducer.js

```diff
@@ -246,7 +264,7 @@ const stepToNextStep = {
 	'setup-wizard-completed': 'summary',
 	'banner-completed': 'woocommerce',
 	'not-started': 'site-type-question',
-	'site-type-question': 'product-suggestions',
+	'site-type-question': 'woocommerce',
```

Comment:
I didn't see anything else about woo in this PR so just making sure this change is intentional.

Reply:
Yes, I wanted to make sure I didn't break any flow and duplicated the value of `product-suggestions`. It falls back to `monitor`.